feat: add tests for extrinsic update_stdlib (#143)
eigerco/pallet-move: tests/assets/move-projects/testing-substrate-stdlib/README.md

# Notes

- Cloned from the original `substrate-stdlib` crate.

> - Keeped the same interface, but replaced native methods by functionless local implementations.
>   Have a look at the [modified balance module](sources/balance.move).

Reviewer: ```suggestion
- The same interface was kept, but functionless local implementations replaced native methods.
  Have a look at the [modified balance module](sources/balance.move).
```

Author: Done
